src/governance/governanceTransactionsFactory.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { assert } from "chai";
+import { Address } from "../core/address";
+import { TransactionsFactoryConfig } from "../core/transactionsFactoryConfig";
+import { GovernanceTransactionsFactory } from "./governanceTransactionsFactory";
+import { Vote } from "./resources";
+
+describe("test multisig transactions factory", function () {
```

Comment:
Typo / multisig.

Reply:
fixed in the next [PR](https://github.com/multiversx/mx-sdk-js-core/pull/616).